lib/plugin/retryTo.js

```diff
@@ -113,6 +113,9 @@ module.exports = function (config) {
 
       recorder.add('retryTo', async () => {
         tryBlock();
+      }).catch(err => {
+        console.error('An error occurred:', err);
+        done(null);
```

Comment:
the test case that i added passed with the resolve promise so i guess it's enough ^^

Reply:
> ah I meant `done()` instead of `done(null)`

The done is the `resolve` of the promise and I get this type 

![Capture d’écran 2024-06-07 à 10 13 01](https://github.com/codeceptjs/CodeceptJS/assets/15615569/4f0613ec-b5bc-4805-ba59-2be2dbcebe81)

and in the file, done function already call with `done(null)` 🤷‍♂️